backend/src/main/java/com/woowacourse/ternoko/domain/member/Coach.java

```diff
@@ -15,20 +15,20 @@ public class Coach extends Member {
     @Column
     private String introduce;
 
-    public Coach(final Long id, final String name, final String nickname, final String email, final String imageUrl,
-                 final String introduce) {
-        super(id, name, nickname, email, imageUrl);
+    public Coach(final Long id, final String name, final String nickname, final String email, final String userId,
+                 final String imageUrl, final String introduce) {
+        super(id, name, nickname, email, userId, imageUrl);
```

Comment:
매개변수를 이런식으로 바꾸면 가독성이 더 좋을 것 같아요!
```suggestion
    public Coach(final Long id,
                 final String name,
                 final String nickname,
                 final String email,
                 final String userId,
                 final String imageUrl,
                 final String introduce) {
        super(id, name, nickname, email, userId, imageUrl);
```

Reply:
생성자에 인자가 많은 클래스는 개행하도록 수정해보았습니다. 확실히 가독성이 좋아지네요.